modules/ROOT/pages/dataweave-selectors.adoc

```diff
@@ -47,44 +41,63 @@ xref:dataweave-cookbook-extract-data#selector_metadata[Extract Data].
 
 |===
 
-== Rules for matching 
+[NOTE]
+====
+The examples below use these selectors. For additional examples, see
+xref:dataweave-cookbook-extract-data#[Extract Data].
+====
 
-All selectors that uses a `name` to select are composed of a `Namespace` and a `Local Part`. When using namespace it is only going to match a `Key` with that `Namespace` and that `Local Part`. If the `name` only has a `Local Part` it is going to match all the `Key` whoes `Local Part` are equal dispaite it has `Namespace` or not.
+== Rules for Matching
 
+Selectors that use a name to select data are composed of a namespace and
+a local part. When relying on the namespace, the selector only matches a key with
+that namespace and that local part. However, if the name only has a local part, the
+selector matches all keys with the same local part regardless of whether
+the name has namespace or not.
 
 
 == Dynamic Selector
 
-The way to do dynamic selection is by using the `[``]` way of selecting. 
-
-- Single Value: `payload[(nameExpression)]`
-- Multi Value: `payload[*(nameExpression)]`
-- Attribute: `payload[@(nameExpression)]`
-- Key Value: `payload[&(nameExpression)]`
+You perform dynamic selection with a `nameExpression` within
+square brackets (`[``]`) that returns a `String` type.
 
-The `nameExpression` should return a `String` type expression.
+* Single Value: `payload[(nameExpression)]`
```

Comment:
Need examples for all Single through Key Value.

Reply:
For the examples, you can replace `(nameExpression)` with `vars.keyName` and explain that `vars.keyName` will be a `String`, such as `age`

Reply:
We're removing the Dynamic Selector section for now due to a bug with dynamically selecting namespaces. (Tested with Shoki)